src/corecel/io/Label.hh

```diff
@@ -50,14 +51,13 @@ struct Label
     //! Create an empty label
     Label() = default;
 
-    //! Create *implicitly* from a C string (mostly for testing)
-    Label(char const* cstr) : name{cstr} {}
-
-    //! Create *implicitly* from just a string name (capture)
+    //! Create *implicitly* from a captured string name (capture)
```

Comment:
Is the different between this constructor documentation and the next constructor intentional?  The other are specifically calling out that the input name is without extension.  Isn't it the case here too?

Reply:
No; I could consolidate them all.

Reply:
Actually the difference is that this one captures whereas the next few copy.

Reply:
That's fair and deserve a callout but the rest of the text should probably be identical.

Reply:
```suggestion
    //! Create *implicitly* from a captured string name without extension (capture)
```